File: CSharp/WcfService1/Bookmark.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace TestAPIns
{
    [DataContract]
    public class Bookmark
    {
        
        public Person Owner;
        [DataMember(Name="id")]
        public long ID;
        [DataMember(Name = "url")]
        public string Url;
        [DataMember(Name = "description")]
        public string Description;

        Bookmark()
        {
            ID = (DateTime.Now.Ticks / TimeSpan.TicksPerMillisecond);
        }
    }
}

File: ruby_sinatra/models/bookmark.rb
class Bookmark
  include MyJson

  attr_accessor :id, :url, :description

  def inititialize()
    id = Time.now.to_i
  end

  def initialize(url, description )
    @id = Time.now.to_i
    @url = url
    @description = description
  end

end

